src/Infrastructure/Controller/Organization/User/DeleteOrganizationUserController.php

```diff
@@ -55,7 +56,8 @@ public function __invoke(Request $request, string $organizationUuid, string $uui
 
         $organization = $organizationUser->getOrganization();
 
-        if (!$this->security->isGranted(OrganizationVoter::EDIT, $organization)) {
+        if (!$this->security->isGranted(OrganizationVoter::EDIT, $organization)
+            || $organizationUser->getRole() === OrganizationRolesEnum::ROLE_ORGA_ADMIN->value) {
```

Comment:
Est-ce qu'on ne devrait pas créer proprement une spec `CanUserEditOrganizationAdmin`...?

Voire `CanUserDeleteOrganizationAdmin`, où on dirait très clairement : il est interdit de supprimer les admins d'une orga (si j'ai bien compris ?)

Reply:
Après je trouverai ça potentiellement étrange. Avoir un pattern `CanUserEditOrganizationAdmin` où on spécifie dedans qu'un rôle admin ne peut pas éditer, ca peut porter à confusion je trouve (on s'attend justement à ce qu'un admin puisse le faire).

Ca ferait quelque chose du genre : 
```php
class CanUserEditOrganizationAdmin
{
    public function isSatisfiedBy(OrganizationUser $organizationUser): bool
    {
        return $organizationUser->getRole() !== OrganizationRolesEnum::ROLE_ORGA_ADMIN->value;
    }
}
```

Reply:
Oui je vois le genre

En relisant je suis moins choqué mais c'est juste que c'est un peu perturbant / difficile à lire à première vue

Dans le cas présent on s'attendrait à un appel du type `if (!$this->canDeleteOrganizationUser(...)) { ... }`

La petite difficulté c'est que les paramètres mélangeraient l'organizationUser, l'organization, et le SymfonyUser... mais si on peut le faire, ça serait mieux, je pense

Reply:
Honnêtement je suis pas sûr que ce soit plus compréhensible que la condition intiiale.

Reply:
Je ne suis pas opposé à laisser comme ça, en relisant c'est OK, comme je disais je pense qu'on s'y fait